src/kubeseal_auto/secrets/sealing.py

```
@@ -0,0 +1,248 @@
+        for secret in secrets:
+            backup_path = secret.with_suffix(secret.suffix + "_backup")
+            output_tmp = secret.with_suffix(secret.suffix + "_new")
+
+            # Create backup of original file
+            shutil.copy2(secret, backup_path)
+
+            ic(cmd)
+
+            try:
+                with secret.open() as stdin_f, output_tmp.open("w") as stdout_f:
+                    subprocess.run(cmd, stdin=stdin_f, stdout=stdout_f, check=True)
```

Ruff (0.14.7)

`subprocess` call: check for execution of untrusted input

(S603)